src/main/resources/static/api.yml

```diff
@@ -3225,3 +3256,22 @@ components:
       enum:
         - Completed
         - Incomplete
+    ActiveOffence:
+      type: object
+      properties:
+        deliusEventNumber:
+          type: string
+          example: "7"
+        offenceDescription:
+          type: string
```

Comment:
perhaps an example value would be helpful here? e.g.

`"Wounding or inflicting grievous bodily harm (inflicting bodily injury with or without weapon) (S20) - 00801"`